config/packages/package-operator/.test-fixtures/affinity-tolerations-resources/package-operator-manager.Deployment.yaml

```diff
@@ -34,9 +34,9 @@ spec:
           containerPort: 8080
         env:
         - name: PKO_SUB_COMPONENT_AFFINITY
-          value: "{\"nodeAffinity\":{\"requiredDuringSchedulingIgnoredDuringExecution\":{\"nodeSelectorTerms\":[{\"matchExpressions\":[{\"key\":\"node-role.kubernetes.io/infra\",\"operator\":\"Exists\"}]}]}}}"
+          value: "{\"nodeAffinity\":{\"requiredDuringSchedulingIgnoredDuringExecution\":{\"nodeSelectorTerms\":[{\"matchExpressions\":[{\"key\":\"hypershift.openshift.io/hosted-control-plane\",\"operator\":\"Exists\"}]}]}}}"
         - name: PKO_SUB_COMPONENT_TOLERATIONS
-          value: "[{\"effect\":\"NoSchedule\",\"key\":\"node-role.kubernetes.io/infra\"}]"
```

Comment:
Why were the labels in this file changed?

Reply:
see above